test/action.yml

```diff
@@ -35,10 +35,10 @@ runs:
   steps:
     - name: Increase JAVA HEAP size
       shell: bash
-      run: echo "JAVA_TOOL_OPTIONS=-Xmx5g -Xms1g" >> $GITHUB_ENV
+      run: echo "JAVA_TOOL_OPTIONS=-Xmx5g -Xms2g" >> $GITHUB_ENV
```

Comment:
Increase the minimum memory here - hope that increases the test suite speed - I didn't benchmark it, just increase it as we have that capacity left on Github Action. If you don't like it, I can revert that back to 1GB 👌